package.json

```diff
@@ -81,6 +81,7 @@
     "rollup-plugin-cleanup": "^3.1.1",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-ts": "^2.0.4",
+    "tslib": "^2.5.0",
```

Comment:
wondering why tslib was added?

Reply:
As per the commit message I was getting `[!] (plugin Typescript) TS2807: This syntax requires an imported helper named '__classPrivateFieldSet' with 5 parameters, which is not compatible with the one in 'tslib'. Consider upgrading your version of 'tslib'.` when running the tests locally.